src/app/projects/project.interface.ts

```diff
@@ -12,8 +12,10 @@ export interface Project {
     raisedFunding: number;
     description: string;
     mainImage: string;
+    images:string[];
```

Comment:
in my opinion, we don't need this one here, because it is part of the events, not the project itself.

and for the slider, we can always load the main image as the first image and then the array of images

Reply:
from what I have understood is that we have to load the project event images in the slider and every project will have only one main image, but it turned out that it is not like that and the project will have multiple images and only those images will be shown in the slider, the event images are not shown in the slider.

in this case RTBM